tests/utils_lib/test_small_fry.py

```diff
@@ -13,25 +13,22 @@ def test_calc_md5():
     assert calc_md5(b"mws") == b"mA5nPbh1CSx9M3dbkr3Cyg=="
 
 
-def test_calc_request_description(cred_access_key, cred_account_id):
+def test_calc_request_description():
     request_description = calc_request_description(
         {
-            "AWSAccessKeyId": cred_access_key,
-            "Markets": cred_account_id,
+            "AWSAccessKeyId": "MY_ACCESS_KEY",
+            "Markets": "MY_ACCOUNT_ID",
```

Comment:
This was a case where using the fixtures for such a tiny test case seemed like overkill. So, changed it to simply check for string literals.